app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
If many arguments are passed to a function or method, then each of its arguments is written on a new line, including the first one.